.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: themattrix/tox
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+    - name: Tests
+      run: |
+          apt-get update -yqq
+          apt-get install shellcheck -yqq
+          shellcheck --shell=bash *.sh
```

Comment:
Also, I would separate this to 3 different steps (`name: Tests ...`), so in case of troubles we easier see if there is error during one of:
1) package installation
2) shellcheck call
3) tox call

Reply:
Changes done.